projects/packages/sync/src/class-sender.php

```diff
@@ -997,5 +998,10 @@ public function uninstall() {
 		// Clear the sync cron.
 		wp_clear_scheduled_hook( 'jetpack_sync_cron' );
 		wp_clear_scheduled_hook( 'jetpack_sync_full_cron' );
+
+		// Drop Sync queue custom table.
+		// Instantiate table storage, so we can drop the custom table. Queue ID is just a placeholder here.
+		$queue_table_storage = new Queue_Storage_Table( 'test_queue' );
```

Comment:
We should also reset the option `custom_queue_table_enabled` to `0` before deleting it, as we want to be able to restore it with the upgrade process if the plugin is re-installed.

If the option is already `1`, we won't go through the init process.